HW3/hw3_app/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from . import views
+
+app_name = 'hw3_app'
+
+urlpatterns = [
+
+    path('', views.index, name='index'),  # вывод главной страницы
+    path('client_orders/', views.client_orders, name='client_orders'),
```

Comment:
Функция `client_orders` должна принимать `id` клиента, но в  маршруте у вас переменной нет. Следовательно функция будет вызывать ошибку и ничего работать не будет.
```suggestion
    path('client_orders/<int:client_id>/', views.client_orders, name='client_orders'),
```
 К тому же? вы не приложили файл БД, а команд на заполнения у вас нет. Проверить функциональность не могу.